api/backend/ecs/load_balancer_manager.go

```diff
@@ -220,6 +220,7 @@ func (e *ECSLoadBalancerManager) listenerToPort(listener *awselb.Listener) model
 	}
 
 	if listener.SSLCertificateId != nil {
+		port.CertificateARN = *listener.SSLCertificateId
 		port.CertificateName = id.CertificateARNToName(*listener.SSLCertificateId)
```

Comment:
won't the diff fail on `port.CertificateName` unless suppressed in terraform?

Reply:
`port.CertificateName` isn't exposed in Terraform. Only `Certificate` which uses `CertificateARN` is it isn't empty.

https://github.com/quintilesims/layer0/blob/develop/plugins/terraform/resource_layer0_load_balancer.go#L293